components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/backchannellogout/LogoutRequestSender.java

```diff
@@ -88,10 +93,26 @@ public void sendLogoutRequests(HttpServletRequest request) {
      * Sends logout requests to all service providers.
      *
      * @param opbsCookieId
+     *
+     * @deprecated This method was deprecated to move OIDCSessionParticipantCache to the tenant space.
+     * Use {@link #sendLogoutRequests(String, String)} instead.
      */
+    @Deprecated
     public void sendLogoutRequests(String opbsCookieId) {
 
-        Map<String, String> logoutTokenList = getLogoutTokenList(opbsCookieId);
+        // For backward compatibility, SUPER_TENANT_DOMAIN was added as the cache maintained tenant.
+        sendLogoutRequests(opbsCookieId, MultitenantConstants.SUPER_TENANT_DOMAIN_NAME);
+    }
+
+    /**
+     * Sends logout requests to all service providers.
```

Comment:
All sp means all sp in particular tenant-right?.... How we handle SAAS applications?

Reply:
Yes. For both sass and non-saas applications we used this method.